apps/liquidity-provision-dashboard/src/app/components/market-list/health-bar.scss

```diff
@@ -0,0 +1,143 @@
+.App {
```

Comment:
chore (blocking): Please use tailwind

suggestion (non blocking): We have a ProgressBar component in the toolkit, which looks quite similar but not exactly the same. Have you considered using it so we can keep things visually consistent? Are there any specific features it needs to support for you to use it?